azure-iot-hub/tests/test_iothub_registry_manager.py

```diff
@@ -0,0 +1,168 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import pytest
+from .device_create_tests import (
+    CreateDeviceWithSymmetricKeyTests,
+    CreateDeviceWithX509Tests,
+    CreateDeviceWithCATests,
+)
+from .device_update_tests import (
+    UpdateDeviceWithSymmetricKeyTests,
+    UpdateDeviceWithX509Tests,
+    UpdateDeviceWithCATests,
+)
+
+from .module_create_tests import (
+    CreateModuleWithSymmetricKeyTests,
+    CreateModuleWithX509Tests,
+    CreateModuleWithCATests,
+)
+from .module_update_tests import (
+    UpdateModuleWithSymmetricKeyTests,
+    UpdateModuleWithX509Tests,
+    UpdateModuleWithCATests,
+)
+
+
+from .common_fixtures import fake_device_id, fake_etag, fake_module_id
+
+
+class DeviceMock(object):
+    @pytest.fixture(scope="function")
+    def mock_device_constructor(self, mocker):
+        return mocker.patch("azure.iot.hub.iothub_registry_manager.Device")
+
+
+class ModuleMock(object):
+    @pytest.fixture(scope="function")
+    def mock_module_constructor(self, mocker):
+        return mocker.patch("azure.iot.hub.iothub_registry_manager.Module")
+
+
+@pytest.mark.describe("IotHub Registry Manager Device Create Tests")
+class TestIotHubRegistryManagerDeviceCreate(
```

Comment:
>TestIotHubRegistryManagerDeviceCreate [](start = 6, length = 37)

when a single test class is inheriting from 3 different functional classes, the pytest output shows only the output for `symmetric key` tests. 
I felt there are 2 ways to have output for all 2 different functional classes that is to repeat 3 tests classes for 3 functional classes. 

1st method :-
`class TestDeviceSas( DeviceMock, CreateDeviceSasTest )`
`class TestDeviceX509( DeviceMock, CreateDeviceX509Test )`

and the 2nd method is to 
rename `CreateDeviceSasTest into TestCreateDeviceSas`
rename `CreateDeviceX509Test into TestCreateDevice`

however if something is wrong in the current implementation let me know